### DISCUSSION TITLE
Convert to discussion feature

### DISCUSSION THREAD

**Discussion by rusty-snake** [netblue30/firejail#4070] [General]
GitHub has a "Convert to discussion" (next to "Lock conversation", "Pin issue", ...). Do we want to use it to move support issues (like #4067 for example) to Q&A? Opinions?

**Comment by reinerh** [comment]
Ah, now I see it too... I already wondered how you moved your tray-icon topic to discussions.

I think we should move everything that is not an "issue" (something that requires code/profile/documentation changes) to discussions.

**Comment by nidamanx** [comment]
I agree to @reinerh
It could be better and faster to manage issues or things supposed to be like that
